easybuild/easyconfigs/c/cairo/cairo-1.14.6-intel-2016a-libX11-1.6.3.eb

```diff
@@ -0,0 +1,56 @@
+easyblock = 'ConfigureMake'
+
+name = 'cairo'
+version = '1.14.6'
+
+libx11 = 'libX11'
+libxver = '1.6.3' 
+versionsuffix = '-%s-%s' % (libx11, libxver)
```

Comment:
@boegel: I am on it, but because the `GLib`-`libxm2` issue, I need a `GLib` with  `Pyhon-libX11`